lib/omnibus/health_check.rb

```diff
@@ -459,15 +484,17 @@ def relocation_check
     def health_check_otool
       current_library = nil
       bad_libs = {}
+      install_name = nil
 
-      yield_shellout_results("find #{project.install_dir}/ -type f | egrep '\.(dylib|bundle)$' | xargs otool -L") do |line|
+      yield_shellout_results("find #{project.install_dir} -type f | egrep '\.(dylib|bundle)$' | xargs otool -L") do |line|
         case line
         when /^(.+):$/
           current_library = Regexp.last_match[1]
-        when /^\s+(.+) \(.+\)$/
+          install_name = get_macos_dylib_install_name(current_library)
+        when /^\s+(.+) \(.+\)?$/
           linked = Regexp.last_match[1]
           name = File.basename(linked)
-          bad_libs = check_for_bad_macos_library(bad_libs, current_library, name, linked)
+          bad_libs = check_for_bad_macos_library(bad_libs, current_library, name, linked, install_name)
```

Comment:
Suggestion: rather than adding a parameter to that function, maybe we can inline the check? Something like

```suggestion
          # Skip check when the "link" is just referring to its install name
          unless install_name && linked == install_name
            bad_libs = check_for_bad_macos_library(bad_libs, current_library, name, linked)
```

Reply:
good idea! I'll double check and do the change

Reply:
done, thanks :)